src/main/java/uk/gov/pay/ledger/util/pagination/PaginationBuilder.java

```diff
@@ -7,6 +7,8 @@
 
 import javax.ws.rs.core.UriInfo;
 import java.net.URI;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
```

Comment:
Again, I think these aren’t needed any more.

Reply:
As above, IDE settings now changed pick this up in the future.